plugins/core/webview/src/q-ui/components/qOptions.vue

```diff
@@ -74,8 +91,32 @@ export default defineComponent({
                 this.$emit('login', new BuilderId())
             } else if (this.selectedLoginOption === LoginIdentifier.ENTERPRISE_SSO) {
                 this.$emit('stageChanged', 'SSO_FORM')
+            } else {
+                // TODO: else ... is not precise
+                // TODO: should pass the entire connection json obj instead of connection id only
+                this.$emit('login', new ExistConnection(this.selectedLoginOption))
+            }
+        },
+        // TODO: duplicates in toolkitOptions, should leverage model/LoginOption interface
+        connectionType(connection: AwsBearerTokenConnection): LoginIdentifier {
+            if (connection.startUrl === SONO_URL) {
+                return LoginIdentifier.BUILDER_ID
             }
+
+            return LoginIdentifier.ENTERPRISE_SSO
         },
+        // TODO: duplicates in toolkitOptions, should leverage model/LoginOption interface
+        connectionTypeDescription(connection: AwsBearerTokenConnection): string {
+            if (connection.startUrl === SONO_URL) {
+                return 'AWS Builder ID'
```

Comment:
Is this displayed anywhere? Does it need to be localized?

Reply:
yea sg